docs/operating-scylla/admin-tools/scylla-sstable.rst

```diff
@@ -26,6 +26,77 @@ The command syntax is as follows:
 
 You can specify more than one SStable.
 
+Schema
+^^^^^^
+All operations need a schema to interpret the SStables with.
+Currently, there are two ways to obtain the schema:
+
+* ``--schema-file`` - Read the schema definition from a file.
+* ``--system-schema`` - Use the known definition of built-in tables (only works for system tables).
+
+By default, the tool uses the first method (``--schema-file``) and it assumes there is a schema file named ``schema.cql`` in the working directory.
```

Comment:
For me it would be clearer if you said:
By default, the tool uses ``--schema-file schema.cql``, i.e., assumes that there is a schema file named ``schema.cql`` in the working directory.